pkg/logger/logger.go

```diff
@@ -0,0 +1,162 @@
+package logger
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"gopkg.in/natefinch/lumberjack.v2"
+)
+
+const (
+	LogFile       = "$HOME/.config/corectl/corectl.log"
```

Comment:
that's fine for now, we should revise where this is as part of the ticket to potentially adopt XDG config